.github/workflows/fuzzing.yml

```
@@ -220,23 +226,32 @@ jobs:
 
+    # If this is a workflow call, run ubpf_fuzzer over each file in the corpus as a regression test.
+    - name: Run fuzzing regression
+      if: inputs.regression_test == true
+      run: |
+        ./ubpf_fuzzer -merge=1 fuzz/corpus new_corpus
+
+    # If this is a scheduled run or a manual run, run ubpf_fuzzer to attempt to find new crashes. Runs for 2 hours.
     - name: Run fuzzing
-      if: matrix.platform == 'ubuntu-24.04'
+      if: matrix.platform == 'ubuntu-24.04' && inputs.regression_test != true
       run: |
```

actionlint

shellcheck reported issue in this script: SC2046:warning:2:154: Quote this to prevent word splitting

(shellcheck)